packages/components-js/src/pages/table.html

```diff
@@ -1,35 +1,39 @@
-<div class="playground light table" title="should render table">
-  <script>
-    const getHeadAndData = async () => {
-      const content = await fetch('assets/table-data-vrt.js').then((res) => res.text());
-      const [, headStr] = /const headVrt = ((\s|\S)*?);/.exec(content) || [];
-      const head = eval(headStr);
+<style>
+  .dark span.table-caption {
```

Comment:
table :host needs to set themed primaryColor, then this won't be necessary due to CSS inheritance.

Reply:
> Table is broken in Safari: <img alt="Screenshot 2023-03-15 at 12 38 27" width="1405" src="https://user-images.githubusercontent.com/30120450/225366193-69ef5e94-2785-4cef-b674-1be803bb4425.png">

Fixed this Safari bug

Reply:
Fixed it